google/cloud/compute/projects/v1/projects.proto

```diff
@@ -102,7 +102,7 @@ service Projects {
   // need. For example, to only include the `id` and `selfLink` fields, add the
   // query parameter `?fields=id,selfLink` to your request.
   // https://cloud.google.com/compute/docs/reference/rest/v1/projects/get
-  rpc GetProjects(GetProjectsRequest)
```

Comment:
There is a `GetXpnResources()` rpc defined a few lines below.  Should that change?